dev/values/hermes/local.yml

```diff
@@ -27,38 +34,41 @@ chains:
       interval: 1s
     gasPrice: 1
     gasDenom: nria
-    trustThreshold: 2/3
+    trustThreshold: 1/3
```

Comment:
this should probably be 2/3 to match tendermint?

Reply:
Updated!
I got these values from https://github.com/astriaorg/hermes/blob/astria/config-astria-celestia.toml
Should I update there as well? Not 100% sure of the implication/meaning of trustThreshold

Reply:
> The trust threshold defines what fraction of the total voting power of a known and trusted validator set is sufficient for a commit to be accepted going forward. - [hermes docs](https://hermes.informal.systems/documentation/commands/path-setup/clients.html?highlight=trust%20threshold#create-client)

Basically 2/3 is the threshold for CometBFT to accept a block so I don't think it makes sense to accept lower than that.